pkg/crypto/internal/wallet/wallet.go

```diff
@@ -194,20 +199,33 @@ func (w *BaseWallet) getKey(verkey string) (*cryptoutil.KeyPair, error) {
 	return &key, nil
 }
 
+// PutKey persists a keypair in the keystore
+func (w *SecretWallet) PutKey(verkey string, pair *cryptoutil.KeyPair) error {
```

Comment:
Probably add a comment for PutKey() and AttachCryptoOperator() are exposed to aid mocking a wallet for testing..

Reply:
PutKey isn't exposed to anything outside pkg/crypto